ml/src/main/scala/ai/h2o/sparkling/ml/algos/ProblemType.scala

```diff
@@ -0,0 +1,43 @@
+package ai.h2o.sparkling.ml.algos
+
+import hex.genmodel.utils.DistributionFamily
+import hex.glm.GLMModel.GLMParameters.Family
+
+object ProblemType extends Enumeration {
+  type ProblemType = Value
+  val Classification, Regression, Both = Value
+
+  def familyToProblemType(family: String): ProblemType = {
+    val enumValue = Family.valueOf(family)
+    enumValue match {
+      case Family.AUTO => Both
```

Comment:
Compilation fail on this line with:
```
[Error] sparkling-water/ml/src/main/scala/ai/h2o/sparkling/ml/algos/ProblemType.scala:13: value AUTO is not a member of object hex.glm.GLMModel.GLMParameters.Family```

Reply:
Nevermind, it is passing on Jenkins when building against H2O in the master branch

Reply:
Yes, this option was recently added to h2o-3. See https://0xdata.atlassian.net/projects/PUBDEV/issues/PUBDEV-7444?filter=allissues&orderby=priority%20DESC&keyword=PUBDEV-7444